src/TraceEvent/TraceEvent.csproj

```diff
@@ -51,6 +51,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Diagnostics.NETCore.Client" Version="$(MicrosoftDiagnosticsNETCoreClientVersion)" />
```

Comment:
This will also require a corresponding addition to `<NuspecProperties>` above, as well as an entry in the dependencies section of the nuspec file.  This is what ensures that NuGet restores the dependency.

Reply:
missed this. it's done now